src/structured-data/structured-data-tags.js

```diff
@@ -96,19 +96,29 @@ function generateArticleData (structuredData = {}, story = {}, publisherConfig =
     "dateCreated": stripMillisecondsFromTime(new Date(story['first-published-at'])),
     "dateModified": stripMillisecondsFromTime(new Date(story['last-published-at'])),
     "name": (storyKeysPresence && story.headline) || '',
-    "image": generateArticleImageData(story['hero-image-s3-key'], publisherConfig)
+    "image": generateArticleImageData(story['hero-image-s3-key'], story, publisherConfig),
+    "isPartOf": generateIsPartOfData(story, publisherConfig)
   }, articleSectionObj(story));
 }
 
-function generateArticleImageData(image, publisherConfig) {
+function generateArticleImageData(image, story = {}, publisherConfig = {}) {
   const articleImage = imageUrl(publisherConfig, image);
 
   return Object.assign({}, {
     "@type": "ImageObject",
-    "url": articleImage
+    "url": articleImage,
+    "image": get(story['hero-image-s3-key'], '')
   }, getQueryParams(articleImage))
```

Comment:
what's `articleImage` and `get(story['hero-image-s3-key'], '')` difference?

Reply:
Article Image is the "cdn-image" from config

Reply:
Oh. Then will the s3 key is an invalid value for the `image`, right? https://schema.org/image says image should be a valid URL. s3 key is not a valid URL I think.

Reply:
Let me see

Reply:
Checked the code. It was added by me, which is not necessary. Tested in the tool and it works fine. So, removed it.

Reply:
Then why add the `story = {}` object as a parameter and changing the current function structure? Remove the older parameters also please.

Reply:
Okay.